### ISSUE TITLE
Implement dynamic roles for server 10.3+ and oCIS

### ISSUE THREAD

**Issue by pmaier1** [owncloud/android#2667]
See Desktop client ticket https://github.com/owncloud/client/issues/7487

**Comment by pmaier1** [comment]
@michaelstingl this needs to be scheduled, thanks!

**Comment by pmaier1** [comment]
Still useful, yes. But it has not made it to any client platform so far and we might take different steps for oCIS. I'd consider it on hold therefore at the moment.

**Comment by michaelstingl** [comment]
Blocked by:
- https://github.com/owncloud/ocis/issues/4848
